docs/concepts/orchestration/scale-out.md

```diff
@@ -55,6 +55,183 @@ executors:
 Flow with three replicas of `slow_encoder` and one replica of `fast_indexer`
 ```
 
+(scale-consensus)=
+## Replicate stateful Executors with consensus using RAFT (Beta)
+
+````{admonition} docarray 0.30
+:class: note
+
+Since docarray version > 0.30, docarray changed its interface and implementation drastically. We intend to support these new docarray versions
+in the near future, but not every feature is yet available {ref}`Check here <docarray-v2>`. This feature has been added with these new docarray versions support.
+
+This feature is only available when using `grpc` as the protocol for the Deployment or when the Deployment is part of a Flow
+````
+
+````{admonition} gRPC protocol
+:class: note
+
+This feature is only available when using gRPC as the protocol for the Deployment or when the Deployment is part of a Flow
+````
+
+Replication is used to scale-out Executors by creating copies of them that can handle requests in parallel providing better RPS.
+However, when an Executor keeps some sort of state, then it is not simple to guarantee that each copy of the Executor keeps the same state,
+which can lead to undesired behavior, since each replica can provide different results, depending on the specific state they hold.
```

Comment:
```suggestion
Replication is used to scale out Executors by creating copies of them that can handle requests in parallel, providing better RPS.
However, when an Executor maintains some sort of state, then it is not simple to guarantee that each copy of the Executor maintains the *same* state,
which can lead to undesired behavior, since each replica can provide different results depending on the specific state they hold.
```